scripts/update-apk.sh

```diff
@@ -53,7 +53,7 @@ if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
     echo "Push to master branch detected, signing the app..."
     cp open-event-master-app-playStore-release-unsigned.apk app-release-unaligned.apk
 	jarsigner -verbose -tsa http://timestamp.comodoca.com/rfc3161 -sigalg SHA1withRSA -digestalg SHA1 -keystore ../scripts/key.jks -storepass $STORE_PASS -keypass $KEY_PASS app-release-unaligned.apk $ALIAS
-	${ANDROID_HOME}/build-tools/27.0.3/zipalign -v -p 4 app-release-unaligned.apk app-release.apk
+	${ANDROID_HOME}/build-tools/27.0.3/zipalign -v -p 4 app-release-unaligned.apk open-event-master-app-playStore-release.apk
```

Comment:
Remaining

Reply:
`cp open-event-master-app-playStore-release-unsigned.apk app-release-unaligned.apk` this will copy and rename the apk and then in last line we are using the latter, what change is remaining? 🤔

Reply:
`app-release-unaligned.apk` And many more

Reply:
okay